examples/cifar10_resnet.py

```diff
@@ -60,6 +93,12 @@
 # Normalize data.
 x_train = x_train.astype('float32') / 255
 x_test = x_test.astype('float32') / 255
+
+if use_pix_mean:
```

Comment:
"subtract_mean_pixel" would be more explicit

Reply:
subtract_pixel_mean variable name is used instead of use_pix_mean.